packages/@react-spectrum/tree/docs/TreeView.mdx

```diff
@@ -397,7 +267,14 @@ You can disable specific items by providing an array of keys to TreeView via the
 
 ```tsx example
 // Using the same tree as above
```

Comment:
mega nit: i don't think we need to have the comment here anymore given that a lot of these examples use `<ExampleTree>` now. same goes for the examples below as well